src/mono/mono/metadata/class-internals.h

```diff
@@ -193,6 +193,9 @@ struct _MonoProperty {
 	MonoMethod *get;
 	MonoMethod *set;
 	guint32 attrs;
+	/* added by metadata-update after class was created;
+	 * not in MonoClassPropertyInfo array - don't do ptr arithmetic */
+	guint32 from_update : 1;
```

Comment:
Can't these be stored in attrs ?

Reply:
@vargaz it's the same as fields - I really don't think there's an unused bit in there.

Reply:
Hm. Actually I'm wrong. Looks like only the low 16 bits have any sort of value (with a big chunk for properties just marked "unused").  So maybe I can put it in the high 16 bits

Reply:
Ok pushed https://github.com/dotnet/runtime/pull/66749/commits/9917ef3c704104da0c6f2e95710e6f7853f7331d which reserves a bit for hot reload.